From 129f75068200da5d39df155387f9993d4ce57ebd Mon Sep 17 00:00:00 2001
From: Brett Jankord <bjankord@gmail.com>
Date: Fri, 15 Sep 2017 11:58:19 -0500
Subject: [PATCH 109/186] README: Document compiling fallback values. (postcss/postcss-custom-properties#75)

I wasn't sure if this was possible with the plugin so I coded it up and it worked. It would be cool to note this in the README so others can save some time by looking at the README.
---
 README.md | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/README.md b/README.md
index 2648f13..3ecb202 100755
--- a/README.md
+++ b/README.md
@@ -59,6 +59,24 @@ div {
 }
 ```
 
+You can also compile CSS custom properties with their fallback value.
+
+Using this `input.css`:
+
+```css
+div {
+  color: var(--color, #f00);
+}
+```
+
+you will get:
+
+```css
+div {
+  color: #f00;
+}
+```
+
 Note that plugin returns itself in order to expose a `setVariables` function
 that allow you to programmatically change the variables.
 
-- 
2.33.1

